x-pack/plugins/alerting/common/routes/r_rule/response/schemas/v1.ts

```diff
@@ -8,40 +8,157 @@
 import { schema } from '@kbn/config-schema';
 
 export const rRuleResponseSchema = schema.object({
-  dtstart: schema.string(),
-  tzid: schema.string(),
+  dtstart: schema.string({
+    meta: {
+      description: 'Rule start date as per UTC.',
+    },
+  }),
+  tzid: schema.string({
+    meta: {
+      description: 'Indicates timezone.',
```

Comment:
>      description: 'Indicates timezone.',

Is this a UTC offset? Or a time zone abbreviation like GMT?

Reply:
As per `packages/kbn-rrule/README.md` it seems like abbreviation 
```typescript
 tzid: string; // Takes a Moment.js timezone string. Recommended to use a country and city for DST accuracy, e.g. America/Phoenix and America/Denver are both in Mountain time but Phoenix doesn't observe DST
```